policy/release/attestation_type_test.rego

```diff
@@ -43,10 +43,16 @@ test_deny_when_pipelinerun_attestation_founds {
 }
 
 test_deny_deprecated_policy_attestation_format {
-	expected := {{
-		"code": "attestation_type.deprecated_policy_attestation_format",
-		"msg": "Deprecated policy attestation format found",
-	}}
+	expected := {
+		{
+			"code": "attestation_type.deprecated_policy_attestation_format",
+			"msg": "Deprecated policy attestation format found",
+		},
+		{
+			"code": "attestation_type.pipelinerun_attestation_found",
+			"msg": "Missing pipelinerun attestation",
```

Comment:
Hey this makes me thing the mocked data is not right. I think you need to wrap the objects of the `attestation` list below in `{"statement": ... }`

Reply:
Oops, good catch. Fixed in my latest push.

Reply:
@lcarva Actually, I believe this is intentional. That test is checking that we spit out an error if we pass in an attestation that ISN'T wrapped with `"statement":`. The test data should reflect that.

I think the extra failure is coming from the fact that the codebase will no longer attempt to correct an improperly formatted attestation. Instead, it'll just fail to recognize the input as an attestation at all, and spit out that additional failure.

Reply:
Aha! You are correct. This makes sense. It sort of makes the `attestation_type.deprecated_policy_attestation_format` redundant, but that's for another day.